src/main/java/com/plaid/client/model/payment/Recipient.java

```diff
@@ -3,14 +3,13 @@
 import com.plaid.client.model.payment.Address;
 
 public final class Recipient {
+  private String recipientId;
   private String name;
   private String iban;
   private Address address;
 
-  Recipient(String name, String iban, Address address) {
```

Comment:
What happened to this constructor?

Reply:
We don't need this because such constructors are not required when the developer will not be instantiating a `Recipient` object. The `Recipient` object is only used as part of the recipient list response schema.